SHiELD/coupler_main.F90

```diff
@@ -17,15 +17,13 @@
 !* License along with FMS Coupler.
 !* If not, see <http://www.gnu.org/licenses/>.
 !***********************************************************************
-
+!> \file
+!> \brief Main driver program for the SHiELD model
+!!
+!! Couples component models for the atmosphere,
+!! ocean (amip), land, and sea-ice using the exchange module
```

Comment:
This is more akin to the simple coupler and doesn't actually have any component compatibility.  It simply sequences the dynamics, radiation/physics, and updates the prognostic state.